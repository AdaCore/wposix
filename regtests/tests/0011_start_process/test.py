from test_support import *

gnatmake('start_proc');
run('start_proc')
