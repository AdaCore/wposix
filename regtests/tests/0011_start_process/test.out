Start_Proc...
Count :  2
 > arg1
 > arg2
Count :  2
 > arg1
 > arg2
Display environment:
OS exists TRUE
Count :  2
 > arg1
 > arg2
Count :  2
 > arg1
 > arg2
Display environment:
START_PROC_1=one
START_PROC_2=two
OS exists FALSE
End Start_Proc
